built_in_tasks/target_capture_task.py

```diff
@@ -284,7 +284,7 @@ class ScreenTargetCapture(TargetCapture, Window):
     plant_visible = traits.Bool(True, desc='Specifies whether entire plant is displayed or just endpoint')
     cursor_radius = traits.Float(.5, desc='Radius of cursor in cm')
     cursor_color = traits.OptionsList("dark_purple", *target_colors, desc='Color of cursor endpoint', bmi3d_input_options=list(target_colors.keys()))
-    cursor_bounds = traits.Tuple((-10., 10., 0., 0., -10., 10.), desc='(x min, x max, y min, y max, z min, z max)')
+    cursor_bounds = traits.Tuple((-18., 18., 0., 0., -10., 10.), desc='(x min, x max, y min, y max, z min, z max)')
```

Comment:
you shouldn't change the default here, rather just change it in your task. otherwise it will affect other new tasks

Reply:
you could also try overriding this variable in the handconstrainedeyecapture class

Reply:
For this part, I just want to change the boundaries for cursor movement, because I may use large boundaries in this task for eye calibration.